notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1566 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    if not os.getenv("IS_TESTING"):
</pre><p>Can you explain to the user why this is exempted during testing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/882/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>It is mentioned in the previous code cell but I'll add a clarifying comment here.</p>